code/style.css

```diff
@@ -1,7 +1,141 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+
+/* first header as flexcontainer, tried using vh to get the font size to adapt to screen */
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: whitesmoke;
+  font-size: 4vh;
+  font-weight: bold;
+}
+
+.menu {
+  font-size: 7vh;
+}
+
+/* hero section */
+.hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100vw;
+  height: 100vh;
+  padding: 2em;
+  box-sizing: border-box;
+  color: whitesmoke;
+  background: url(https://mir-s3-cdn-cf.behance.net/project_modules/1400_opt_1/44ed8b50022933.58c566bd36679.gif) center center no-repeat;
+  background-size: cover;
+}
+
+@media only screen and (max-width: 667px) { .hero { background: url(https://mir-s3-cdn-cf.behance.net/project_modules/1400_opt_1/44ed8b50022933.58c566bd36679.gif) center center no-repeat; background-size: cover; } }
+
+
+.hero-title {
+  max-width: 16em;
+  margin: 0;
+  font-size: 10vh;
+  font-weight: bold;
+  font-family: 'Fraunces', serif;
+}
+
+@media only screen and (min-width: 667px) { .hero-title { font-size: 14vh; } }	
+
+.hero-footer {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+.hero-form {
+  text-align-last: center;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  border: 2px inset whitesmoke;
+  border-radius: 2px;
 }
+
+.hero-labels {
+  padding: 5px;
+  color: whitesmoke;
+  font-size: 14px;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Fraunces', serif;
+}
+
+input {
+  margin-top: 3px;
+  margin-bottom: 3px;
+
+}
+
+.submit-button {
+	color: #444;
+  text-decoration: none;
+  margin: 3px;
+}
+
+
+/* accordion style and functionality*/
+
+.accordion {
+  font-family: 'Fraunces', serif;
+}
+
+.did-you-know {
+  background-color: #eee;
+  color: #444;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
```

Comment:
These two together makes the content grow too wide:
<img width="357" alt="Skärmavbild 2021-02-09 kl  17 13 03" src="https://user-images.githubusercontent.com/51529201/107392539-13b61380-6afa-11eb-9a9e-2bd91c26d2e9.png">